elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/indexes/MappingHandlers.scala

```diff
@@ -15,7 +15,10 @@ trait MappingHandlers {
           val raw = ResponseHandler.fromResponse[Map[String, Map[String, Map[String, Any]]]](response)
           val raw2 = raw.map {
             case (index, types) =>
-              val mappings = types("mappings").getOrElse("properties", Map.empty)
+              val mappings = types("mappings") match {
+                case Map.empty => Map.empty
```

Comment:
In 7.1.1, if there are no mappings, the map is empty so rather than using the existence of "properties", just test to see if it's an empty map.